js/islandora_image_annotation_dialog.js

```diff
@@ -316,9 +316,21 @@
         svgEncoding.setAttribute('content', 'utf-8');
         svgAbout.appendChild(svgEncoding);
       });
-      // XXX: Need to handle the output this way because of IE browser
-      // incompatabilities.
-      return $(xmldoc.documentElement).html();
+      
+      // Test browser features and call correct to string method
+      var serializeXmlComp = function(xmldoc) {
+        if (typeof window.XMLSerializer != "undefined") {
+          return (new window.XMLSerializer()).serializeToString( xmldoc );
+        } else if (typeof xmldoc.xml != "undefined") {
+          return xmldoc.xml;
+        }
+        return "";
+      }
+ 
+      // convert XML DOM document to string
+      var xmlAsString = serializeXmlComp( xmldoc );
+
+      return xmlAsString;
```

Comment:
Hi @nigelgbanks  thanks for reviewing this.  innerHTML is not (standard) available for node elements when handling XML , so html() is not reliable. Iexplorer 8 was able to handle this(i think), but since 9 they started standardising (so the XMLSerializer is called on those).

Jquery documentation states the method is not present for XML. https://api.jquery.com/html/. I checked with Multiple Chrome, Safari and Firefox versions and having inconsistent results with .innerHTML i went for this solution. Hope this fill stick for long time.